### DISCUSSION TITLE
FNIRSI GC01: Install with stlink dongle

### DISCUSSION THREAD

**Discussion by webrex2000** [Gissio/radpro#161] [General]
I can't see the Geiger counter in my pc win 10 then I saw instructions regarding stlink Dongle installation. The 4 pins have labels different from the instructions. From left to right I see clk dio gnd vcc....... I hope this is not a problem. What does mean to activate the bat file? Do I need to just  double click the bat  file and installation will start?

**Comment by ChyrkaA** [comment]
It should be obvious.
3.3v to VCC
GND to GND
SWDIO to DIO
SWCLK to CLK

The square hole by the battery is VCC

**Reply by webrex2000** [reply]
Yes, in the Gissio instructions the pin  order is different from my device.

**Comment by ot4eto78** [comment]
![462582453_940136137512021_5587910172528392738_n](https://github.com/user-attachments/assets/e8b07c74-819d-4ab3-9f87-6ef85371fbc6)

**Reply by webrex2000** [reply]
Yes is like the picture.... Good. Thank you Last question.... I'm not English speaking..... In the instruction  I see I must activate an install bat file...... Activate means I need to double click the file to install the firmware or I need to do something different?

**Reply by denka977** [reply]
yes, if the contacts are connected correctly (according to the inscriptions on your dosimeter board), then stlink will give a green or red LED signal. Then run the .bat file on your computer and stlink will blink like a Christmas tree (green-red), the firmware progress will be available in the command line window (running line). Wait for some time, then your dosimeter will turn on itself and start clicking (informing about the gamma quantum that has flown in). Yes, it turns on itself even without a connected battery, because it gets power from stlink.

**Reply by webrex2000** [reply]
Ok I just double click the bat file and let see the show:) thank you

**Reply by ot4eto78** [reply]
It worked for me. I just need to set it up.

**Reply by webrex2000** [reply]
Ok i' m waiting for the Dongle....got it at aliexpress..... Hope to solve my problem.... My counter does not have the j321 tube..... I need to upgrade firmware and USB does not work...let' try the Dongle .. Never ending story!

**Reply by webrex2000** [reply]
Hi i just updated the firmware and changed to the j321 tube, I used custom Hv profile but I have 91 volts on the tube, is this normal?

**Comment by webrex2000** [comment]
Hi i just updated the firmware and changed to the j321 tube, I used custom Hv profile but I have 91 volts on the tube, is this normal?

**Reply by soosp** [reply]
You can't measure the tube voltage directly with an ordinary voltmeter, because it have too low internal resistance (usually 10 MOhm). You need a 1 GOhm resistor in series with your meter and have to calculate the tube voltage.
